File: PHP/TokoElektronik.php
<?php

// kelas untuk menyimpan data produk
class TokoElektronik{
    private int $id;
    private string $nama;
    private string $merk;
    private int $harga;
    private int $stok;
    private string $gambar;


    public function __construct(int $id, string $nama, string $merk, int $harga, int $stok, string $gambar){
        $this->id = $id;
        $this->nama = $nama;
        $this->merk = $merk;
        $this->harga = $harga;
        $this->stok = $stok;
        $this->gambar = $gambar;

    } 

    //getter dan setter unutuk akses
    public function getId(): int{
        return $this->id;
    }

    public function setID(int $id): void{
        $this->id = $id;
    }

    public function getNama(): string{
        return $this->nama;
    }

    public function setNama(string $nama): void{
        $this->nama = $nama;

    }

    public function getMerk(): string{
        return $this->merk;
    }

    public function setMerk(string $merk): void{
        $this->merk = $merk;
    }

    public function getHarga(): int{
        return $this->harga;
    }

    public function setHarga(int $harga): void{
        $this->harga = $harga;
    }

    public function getStok(): int{
        return $this->stok;
    }

    public function setStok(int $stok): void{
        $this->stok = $stok;
    }

     public function getGambar(): string{
        return $this->gambar;
    }

    public function setGambar(string $gambar): void{
        $this->gambar = $gambar;
    }


    


}

File: Java/TokoElektronik.java
public class TokoElektronik{
   //atribut dari kelas
     private int id;
     private String namaProduk;
     private String merk;
     private int harga;
     private int stok;


     public TokoElektronik(int id, String namaProduk, String merk, int harga, int stok){


        this.id = id;
        this.namaProduk = namaProduk;
        this.merk = merk;
        this.harga = harga;
        this.stok = stok;

     }

     //getter & setter

     public int getId(){
      return id;
     }
     public void setId(int id){
      this.id = id;
     }


     public String getNamaProduk(){
        return namaProduk;
     }

     public void setNamaProduk(String namaProduk){
        this.namaProduk = namaProduk;
     }

     
     public String getMerk(){
        return merk;
     }

     public void setMerk(String merk){
        this.merk = merk;
     }

     
     public int getHarga(){
        return harga;
     }

     public void setHarga(int harga){
        this.harga = harga;
     }

     public int getStok(){
        return stok;
     }

     public void setStok(int stok){
        this.stok = stok;
     }

    
     public void tampilkan(){
    System.out.printf("| %-3d | %-15s | %-12s | %-8d | %-4d |\n", id, namaProduk, merk, harga, stok);




     }


     

     







}